wormhole/file_transport.go

```diff
@@ -484,11 +504,11 @@ func (t *fileTransport) listenRelay() (err error) {
 	ctx := context.Background()
 
 	var conn net.Conn
-	switch t.relayURL.Proto {
+	switch t.relayURL.Scheme {
 	case "tcp":
 		// NB: don't dial the relay if we don't have an address.
-		addr := t.relayURL.Addr()
-		if addr == ":0" {
+		addr := t.relayURL.Host
```

Comment:
minor nitpick. May be the name `addr` should be just `hostname` or something?

Reply:
No, actually this is a bug and the full address should be used.